tests/util/foreign-chains.ts

```diff
@@ -0,0 +1,207 @@
+export interface ForeignChainsInfo {
+  moonbeamNetworkName: string;
+  moonbeamParaId: number;
+  foreignChains: ForeignChainInfo[];
+}
+
+export interface ForeignChainInfo {
+  name: string;
+  paraId: number;
+  muted?: boolean;
+  endpoints: string[];
+}
+
+export const ForeignChainsEndpoints: ForeignChainsInfo[] = [
+  {
+    moonbeamNetworkName: "Moonriver",
+    moonbeamParaId: 2023,
+    foreignChains: [
+      {
+        name: "Statemine",
+        paraId: 1000,
+        endpoints: [
```

Comment:
Shouldn't we use the endpoints of polkadotjs instead ? they are more likely to be maintained ?

Reply:
Done